webapp/src/websocket.ts

```diff
@@ -7,7 +7,7 @@ import {logWarn, logErr} from './log';
 
 export default class WebSocketClient extends EventEmitter {
     private ws: WebSocket | null;
-    private seqNo = 0;
+    private seqNo = 1;
```

Comment:
just curious what this is for?

Reply:
The change was to conform to what's expected on the server side as the zero value is treated as invalid. I found this during debugging but it doesn't affect anything at the moment. The sequence number is used by the server to figure out which events to resend to the client in case of an unexpected reconnection.

Reply:
Sorry, went into "teaching mode" there :sweat_smile: